UNRELEASED.md

```diff
@@ -31,3 +31,5 @@ See [`CHANGELOG.md`](CHANGELOG.md) for real-world examples of good changelog ent
 - `bpk-component-horcrux`:
   - Fixed issue where `BpkHorcrux` would occasionally possess the living.
 
+- `bpk-component-dialog`:
```

Comment:
Can you move these to under the `Unreleased` header please? This the example section